src/Microsoft.SqlTools.ServiceLayer/Metadata/MetadataScriptTempFileStream.cs

```diff
@@ -0,0 +1,99 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Metadata
+{
+    public static class MetadataScriptTempFileStream
+    {
+        private const string DirectoryName = "TableAndViewScripts";
+
+        public static void Write(string serverName, IEnumerable<string> scripts)
+        {
+            var tempFileName = $"{DirectoryName}/{serverName}.tmp";
```

Comment:
use path.combine here to avoid any slash direction problems on different platforms